1.chunkedgraph.md

```diff
@@ -64,6 +64,8 @@ and link the image and tag in `values.yaml` (refer to `example_values.yaml`).
 
 > NOTE: Depending on your dataset, you will need to figure out the optimal limits for cpu and memory in your worker deployments. To do that adjust the `count` and `machine` variables in terraform.tfvars. It can vary with chunk size, size of supervoxels (atomic semgents in layer 1), number of edges per chunk and so on.
 
+> NOTE: The bucket storing the edges, components etc must be public. The bucket is also linked in `example_values.yaml` [here](https://github.com/seung-lab/CAVEpipelines/blob/4d0ee8d98fb8891074d6e1115ed9394e6a021cd5/helm/pychunkedgraph/example_values.yaml#L29).
```

Comment:
The bucket with segmentation data needs to be public, not the one with edges and components. They do not need to be in the same bucket. It needs to be public for neuroglancer to read.